definitions/npm/ramda_v0.x.x/flow_v0.62.x-/test_ramda_v0.x.x_list.js

```diff
@@ -147,12 +148,34 @@ const str: string = "hello world";
   const findxs6: number = _.findLastIndex(_.propEq("a", 2), os);
   const findxs7: number = _.findLastIndex(_.propEq("a", 4))(os);
 
-  const s: Array<number> = filter(x => x > 1, [1, 2]);
-  const s1: Array<string> = _.filter(x => x === "2", ["2", "3"]);
-  const s3: { [key: string]: string } = _.filter(x => x === "2", {
-    a: "2",
-    b: "3"
-  });
+  describe('filter', () => {
+    it('perserves the element type in the predicate and result array (number)', () => {
+      const s: Array<number> = filter(x => x > 1, [1, 2]);
+    })
+
+    it('perserves the element type in the predicate and result array (number)', () => {
+      const s: Array<string> = _.filter(x => x === "2", ["2", "3"]);
+    })
+
+    it('filters objects by passing the value to the predicate', () => {
+      const s: { [key: string]: string } = _.filter(x => x === "2", {
+        a: "2",
+        b: "3"
+      });
+    })
+
+    it('refines the element type using the predicate', () => {
+      const notNull = (x): bool %checks => x != null
```

Comment:
Check the reverse, that it errors when `RefineFilter` is not applied.

Reply:
Good catch! Done.